lifecycle/README.rst

```diff
@@ -63,19 +59,19 @@ We split the tasks of the talker node into separate pieces and execute them as f
 #. cleanup: We destroy the publisher and timer
 
 The principle is implemented in this demo as the typical talker/listener demo.
-However, imaging a real scenario with attached hardware which may have a rather long booting phase, i.e. a laser or camera.
-One could image bringing up the device driver in the configuring state, start and stop only the publishing of the device's data and only in the cleanup/shutdown phase actually shutdown the device.
+However, imagine a real scenario with attached hardware which may have a rather long booting phase, i.e. a laser or camera.
+One could imagine bringing up the device driver in the configuring state, start and stop only the publishing of the device's data in active/deactive state, and only in the cleanup/shutdown state actually shutdown the device.
 
 The ``lifecycle_listener`` is a simple listener which shows the characteristics of the lifecycle talker.
-The talker enables the message publishing only in the active state and thus making the listener receiving only messages when the talker is in an active state.
+The ``lifecycle_talker`` enables message publishing only in the active state and thus the listener only receives messages when the talker is in an active state.
```

Comment:
An aside: unfortunately, subscribers are not lifecycle enabled. This creates a bit of an issue because a lifecycle node that is not yet in the Active state can receive messages and so has to implement its own mechanism to avoid receiving messages when not in the Active state. This can happen when lifecycle nodes are mixed with other nodes (publishers) that are not lifecycle enabled.

Reply:
Agreed.  That problem is tracked somewhere else (I can't find the ticket at the moment), but I'm not sure we should update anything here in the documentation about that.  We'll obviously have to update this once we implement a subscriber state machine.